tools/image-info

```diff
@@ -2685,12 +2685,20 @@ def append_partitions(report, image):
                 jsonschema.validate(options, info.get_schema()["properties"]["options"])
 
                 # Finally mount
-                mmgr.mount(mounts.Mount(
-                    part_device,
-                    info,
-                    devices_map[part_device],  # retrieves the associated Device Object
-                    part_mountpoint,
-                    options))
+                mnt_kwargs = {
+                    "name": part_device,
+                    "info": info,
+                    # retrieves the associated Device Object
+                    "device": devices_map[part_device],
+                    "target": part_mountpoint,
+                    "options": options
+                }
+                # XXX: remove inspect and just add once osbuild
```

Comment:
nitpick: this looks like an incomplete comment... :thinking:

Reply:
Thanks, nice catch! fixed now